docs/getting-started/setup/getting-and-setting-data.md

```diff
@@ -8,40 +8,60 @@ order: 10
 
 # Getting and setting data
 
-CKEditor 5 allows you to retrieve the data from and save it to your server (or to your system in general) in various ways. In this guide, you can learn about the available options along with their pros and cons.
+CKEditor&nbsp;5 allows you to retrieve the data from and save it to your server (or to your system in general) in various ways. In this guide, you can learn about the available options along with their pros and cons.
 
-## Manually retrieving the data
+## Autosaving
 
-When you:
+One of the best ways to enhance the user experience in saving editor data is by providing an autosave functionality. This ensures users do not need to remember to save their data, and it prevents your system from being flooded with save events since the save action will be debounced.
```

Comment:
```suggestion
One of the best ways to improve the user experience is to automatically save editor data as it changes. This ensures that users do not have to remember to save their data and prevents work loss.
```

IMO we don't need the information about debouncing, as it is an implementation details and is already mentioned in the linked document.